tests/run_wgrib2_aec_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# This is an optional test script for the wgrib2 project.
+#
```

Comment:
For  each test script add at least a one-sentance comment starting with "This test checks..."

So something like:

This checks the use of AEC compression, only available if USE_AEC is turn on in CMake.